### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Comment by Daniil-Pankieiev** [comment]
image
image
image
image
image
image
image
image

**Review by Wonsky1 — APPROVED** [review]

**Review by NEkropo1 — CHANGES_REQUESTED** [review]
Add new lines everywhere(including jsons)

File: frontend/src/views/MovieDetailsScreen.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default {
 
     async fetchMovie (id) {
       try {
-        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}`, {
+        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies/${id}`, {
```
Comment (NEkropo1) [inline frontend/src/views/MovieDetailsScreen.vue:45]: const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies/${id}/`

File: frontend/src/views/MovieSessionDetailsScreen.vue [hunk 2]
```diff
@@ -68,7 +68,7 @@ export default {
     async fetchMovieSession (id) {
       try {
         this.loading = true;
-        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}`, {
+        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions/${id}`, {
```
Comment (NEkropo1) [inline frontend/src/views/MovieSessionDetailsScreen.vue:71]: const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions/${id}/`

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]
you didn't fix Paul's comment:
`Add new lines everywhere(including jsons)`

**Comment by Daniil-Pankieiev** [comment]
> you didn't fix Paul's comment: `Add new lines everywhere(including jsons)`

Sorry, forgot about this. Fixed it

**Pull request closed without merging**